### PR TITLE
[Auckland-2017] Speakers Update

### PR DESCRIPTION
**Pull request by DefSol**
Mainly adding speaker artifacts, but also includes some touch ups stating our Early Bird has close. This has also been rebased against the branch that Flavia has just submitted for some new sponsors.

### CODE REVIEW

File: content/events/2017-auckland/program/sanjeev-sharma.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
++++
+Talk_date = ""
+Talk_start_time = ""
+Talk_end_time = ""
+Title = "When DevOps met SRE: From Apollo 13 to Google SRE:"
```
Comment (adrianmoisey) [inline content/events/2017-auckland/program/sanjeev-sharma.md]: There's a trailing `:` here. Not sure if that's intentional.

**Comment by DefSol** [comment]
Nice spot - thanks for that - have amended and just got another another head shot from one of our speakers - so added this as well.

**Comment by phrawzty** [comment]
Hello! I see that there has been some movement since the initial PR. Is this ready to merge? No pressure - I just want to make sure before I hit the button. :)

**Comment by bridgetkromhout** [comment]
> Hello! I see that there has been some movement since the initial PR. Is this ready to merge? No pressure - I just want to make sure before I hit the button. :)

It's 3:29am there now. So I don't think they're going to update it more tonight. :)

**Pull request merged**